src/libraries/Microsoft.Extensions.Hosting/tests/UnitTests/AssemblyInfo.cs

```diff
@@ -2,6 +2,8 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 using System;
+using Microsoft.Extensions.Configuration.UserSecrets;
 using Xunit;
 
 [assembly: ActiveIssue("https://github.com/dotnet/runtime/issues/37669", TestPlatforms.Browser)]
+[assembly: UserSecretsId("Microsoft.Extensions.Hosting.Unit.Tests")]
```

Comment:
Is this necessary?

Reply:
Afaik assembly must have this attribute
https://github.com/dotnet/runtime/blob/6072e4d3a7a2a1493f514cdf4be75a3d56580e84/src/libraries/Microsoft.Extensions.Configuration.UserSecrets/src/UserSecretsConfigurationExtensions.cs#L131-L135

Reply:
Got it. Yes that appears that's the only way Hosting will hook up UserSecrets:

https://github.com/dotnet/runtime/blob/eb187d1250cf65c16442c207ff97da65690afae5/src/libraries/Microsoft.Extensions.Hosting/src/Host.cs#L81-L84